pinot-common/src/main/java/org/apache/pinot/common/utils/request/RequestUtils.java

```diff
@@ -174,10 +176,23 @@ public static Expression getLiteralExpression(String value) {
 
   public static Expression getLiteralExpression(byte[] value) {
     Expression expression = createNewLiteralExpression();
+    // TODO: Set the binary value directly instead of converting it to hex string.
```

Comment:
(minor) Add comment stating this is for backward compatibility, and we can switch one release after server side can directly use binary